test/bad_dtls_test.c

```diff
@@ -306,12 +306,9 @@ static int send_record(BIO *rbio, unsigned char type, uint64_t seqnr,
     memcpy(enc, msg, len);
 
     /* Append HMAC to data */
-    if ((hmac = EVP_MAC_fetch(NULL, "HMAC", NULL)) == NULL)
-        return 0;
-    ctx = EVP_MAC_CTX_new(hmac);
-    EVP_MAC_free(hmac);
-    if (ctx == NULL)
-        return 0;
+    if ((hmac = EVP_MAC_fetch(NULL, "HMAC", NULL)) == NULL
```

Comment:
... not many TEST_()  in here ... oh well.

Reply:
Added TEST_ptr around these.  This test isn't great in this regards.